package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "webpack": "NODE_ENV=production webpack -p --bail",
     "watch": "webpack --watch",
+    "prestart": "npm install",
```

Comment:
Since, we are mounting the folder for development, we need to reinstall the node_modules. I didn't want to have a separate script which does `npm install && npm start`, so I used `prestart`